extensions/2.0/Khronos/KHR_materials_volume_scatter/schema/glTF.KHR_materials_volume_scatter.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_materials_volume_scatter glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines the scattering parameters for the volume of a material.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "multiscatterColor": {
+            "type": "array",
+            "items": {
+                "type": "number",
+                "minimum": 0.0,
+                "maximum": 1.0
+            },
+            "description": "The multi-scatter albedo.",
+            "default": [ 0.0, 0.0, 0.0 ],
+            "minItems": 3,
+            "maxItems": 3
+        },
+        "scatterAnisotropy": {
+            "type": "number",
+            "minimum": -1.0,
+            "maximum": 1.0,
+            "default": 0.0,
+            "description": "The anisotropy of scatter events."
```

Comment:
Can we add a description (either to this directly, or with the `gltf_detailedDescription` pattern [used elsewhere](https://github.com/KhronosGroup/glTF/blob/90139f8f126c222a4b684219f226f85004ad45fc/extensions/2.0/Khronos/KHR_materials_volume/schema/material.KHR_materials_volume.schema.json#L25)?

It could say something like "The anisotropy of scatter events. Range is (-1, 1).  1 indicates forward scattering, -1 indicates backscattering, and the default is zero."

Did I get that backwards?  We should spell it out here.